extra/2-piping.js

```diff
@@ -16,26 +16,26 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+   return a + b;
 }
 
-function multiply() {
-
+function multiply(a, b) {
+   return a * b;
 }
 
-function format() {
-
+function format(a) {
+   return `£${a}`;
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+let badCode = `£${(startingValue + 10) * 2}`;
 
 /* BETTER PRACTICE */
 
-let goodCode =
+let goodCode = format(multiply(startingValue + 10, 2));
```

Comment:
you've probably missed `add()` function :)
also in some cases with heavy code it can be better practice to write more readable code this way:
```js
let sum = add(startingValue, 10);
let product = multiply(sum, 2);
let result = format(product);
```